values.yaml

```diff
@@ -235,6 +240,9 @@ nginx:
   image:
     repository: goharbor/nginx-photon
     tag: dev
+    # Override default imagePullSecrets
```

Comment:
Is it necessary to provide a secret for each image? In what scenario this is needed?

Reply:
I am using public registries like hub.docker.com or quay.io along with a locally hosted for my own developments.
I do not want to see the list of all secrets in every deployment, but this is more to my own taste.
It is definitely not necessary.
Do you want it removed?

Reply:
I think that keep only one overall config is better.